data_structures/stack_using_queue.cpp

```diff
@@ -0,0 +1,108 @@
+/**
+ * @file
+ * @brief Implement a last in first out (LIFO) stack using queues.
+          The implemented stack should support all the functions of a normal queue (push, top, pop, and empty).
+ * @author [Neha Labhasetwar](https://github.com/nehalabhasetwar)
+ */
+
+#include <iostream>  // for standard IO operations
+#include <cassert>   // defines one macro function (assert) that can be used as a standard debugging tool
+#include <queue>     // This header is part of the containers library, designed to operate in FIFO context
+
+class MyStack
```

Comment:
Add a brief description of what class does/works for.
```suggestion
/**
 * @brief description
 */
class MyStack
```